radio/src/telemetry/frsky.h

```diff
@@ -139,8 +139,8 @@
 #define RBOX_STATE_LAST_ID        0x0b2f
 #define RBOX_CNSP_FIRST_ID        0x0b30
 #define RBOX_CNSP_LAST_ID         0x0b3f
-#define DIY_FIRST_ID              0x5000
-#define DIY_LAST_ID               0x52ff
```

Comment:
Ah so you want to have a complete DIY range (not used in code) and a Streaming part